src/odemis/driver/test/andorcam2_test.py

```diff
@@ -78,24 +76,26 @@ class TestFake(VirtualTestCam, unittest.TestCase):
     camera_type = CLASS_SIM
     camera_kwargs = KWARGS_SIM
 
-    def setUp(self):
+    @classmethod
+    def setUpClass(cls):
         # Ignore RuntimeWarning: numpy.ndarray size changed, may indicate binary incompatibility.
         # Expected 80 from C header, got 88 from PyObject
         # This warning is not caused by the code explicitly changing the array size but rather
         # by an inconsistency between different versions of NumPy.
         warnings.filterwarnings(
             "ignore", category=RuntimeWarning, message=re.escape("numpy.ndarray size changed")
         )
+        cls.camera = cls.camera_type(**cls.camera_kwargs)
```

Comment:
That's what super() does, right? So just replace this line by a call to `super()`. (and preferably place it as first line of the function).